src/content/blog/welcome-to-the-forest-with-chainsafe-s-filecoin.md

```diff
@@ -19,9 +19,9 @@ Filecoin Mainnet launched one year ago this week, opening up the decentralized s
 
 Implementations are a critical component of protocol infrastructure which allow developers, clients, and storage providers to securely connect to and build on the blockchain. Forest now joins Lotus, the reference implementation written in Go, as the second implementation devnet available within the Filecoin ecosystem.
 
-Built by the team at Chainsafe, this first-iteration of Forest is an alpha release that is now fully synced with the various [protocol enhancements and network upgrades](/fips/) that have been delivered in recent months. As a next step, developers are invited to install, test, and run the Forest client in order to generate feedback and future use-cases.
+Built by the team at Chainsafe, this first-iteration of Forest is an alpha release that is now fully synced with the various protocol enhancements and network upgrades that have been delivered in recent months. As a next step, developers are invited to install, test, and run the Forest client in order to generate feedback and future use-cases.
```

Comment:
`fips` should redirect `/blog/participating-in-the-filecoin-ecosystem-bounties-microgrants-and-fips`

Flagging in case you missed it, but maybe that link is irrelevant with the content and is rightfully removed.

Reply:
I missed it. Thank you.